browser/app/js/objects/__tests__/actions.test.js

```diff
@@ -221,20 +206,18 @@ describe("Objects actions", () => {
     const expectedActions = [
       {
         type: "objects/SET_SORT_BY",
-        sortBy: "name"
+        sortBy: SORT_BY_NAME
       },
       {
         type: "objects/SET_SORT_ORDER",
-        sortOrder: true
+        sortOrder: SORT_ORDER_ASC
       },
       {
         type: "objects/SET_LIST",
-        objects: [],
-        isTruncated: false,
-        marker: ""
```

Comment:
All the frontend tests have ran fine, but as we don't use `marker` anymore, should we reflect that fact while creating the mock store on line 197 of this file as well?

Reply:
Also, we can remove `istruncated` and `nextmarker` from `browser/app/js/objects/__tests__/actions.test.js:46-47` as we don't use those variables anymore while listing objects.

Reply:
Done